src/chi-vue/src/components/data-table-bulk-actions/DataTableBulkActions.tsx

```diff
@@ -0,0 +1,149 @@
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { findComponent } from '@/utils/utils';
+import {
+  BULK_ACTIONS_CLASSES,
+  BUTTON_CLASSES,
+  CHECKBOX_CLASSES,
+  CLOSE_CLASS,
+  DIVIDER_CLASSES,
+  FORM_CLASSES,
+  ICON_CLASS,
+  ICON_CLASSES,
+  UTILITY_CLASSES,
+} from '@/constants/classes';
+import DataTable from '../data-table/DataTable';
+
+@Component({})
+export default class DataTableBulkActions extends Vue {
+  @Prop() bulkActionConfig!: any;
+
+  showBulkActions?: boolean = true;
+
+  _selectedRows(e: any) {
+    this.$emit('chiShowSelectedOnly', e.srcElement.checked);
+  }
+
+  mounted() {
+    const dataTableComponent = findComponent(this, 'DataTable');
+
+    if (dataTableComponent) {
+      (dataTableComponent as DataTable)._bulkActionsComponent = this;
+    }
+  }
+
+  toggleCloseBulkActions() {
+    this.$props.bulkActionConfig.selectedRow = false;
+  }
+
+  render() {
+    return (
+      <transition name="slide-fade">
+        {this.$props.bulkActionConfig.selectedRow.length > 0 && this.$props.bulkActionConfig.selectedRow && (
+          <div class={`${BULK_ACTIONS_CLASSES.BULK_ACTIONS}`}>
+            <div class={`${BULK_ACTIONS_CLASSES.BULK_ACTIONS_TOP} ${UTILITY_CLASSES.Z_INDEX.Z_40}`}>
+              <chi-link href="#">Select all</chi-link>
+              <chi-link href="#">Clear</chi-link>
+            </div>
+            <div class={`${BULK_ACTIONS_CLASSES.BULK_ACTIONS_START}`}>
+              <div class={`${BULK_ACTIONS_CLASSES.BULK_ACTIONS_RESULTS}`}>
+                <div class={`${BULK_ACTIONS_CLASSES.BULK_ACTIONS_LABELS}`}>
+                  Actions ({this.bulkActionConfig.selectedRow.length} Selected)
+                </div>
+                <div class={`${FORM_CLASSES.FORM_ITEM}`}>
+                  <div class={`${CHECKBOX_CLASSES.checkbox}`}>
+                    <input
+                      id={`checkbox-ba${this.bulkActionConfig.selectedRow.length}`}
```

Comment:
What if there are 2 data tables on the same page with the same amount of selected rows? In certain flows, there might be a duplication of IDs